.circleci/config.yml

```diff
@@ -56,6 +55,101 @@ jobs:
 
 
 
+  build-linux-release:
+    <<: *defaults
+    docker:
+      - image: hyperledger/iroha-docker-develop
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - build-cache-release-{{ arch }}-{{ .Branch }}-
+            - build-cache-release-{{ arch }}-
+          paths:
+            - ~/.ccache
+      - run:
+          name: cmake
+          command: >
+            cmake \
+              -DCOVERAGE=OFF \
+              -DTESTING=OFF \
+              -H$IROHA_HOME \
+              -B$IROHA_BUILD \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DPACKAGE_DEB=ON \
+              -DPACKAGE_TGZ=ON
+      - run:
+          name: make and package
+          command: |
+            cmake --build $IROHA_BUILD --target package -- -j4
+      - save_cache:
+          key: build-cache-release-{{ arch }}-{{ .Branch }}-{{ epoch }}
+          paths:
+            - ~/.ccache
+      - run:
+          name: copy deb file
+          command: |
+            cd $IROHA_BUILD
+            cp iroha-*.deb ./iroha.deb
+            cp iroha-*.tar.gz ./iroha.tar.gz
+            # if IROHA_VERSION is not set, then CIRCLE_BUILD_NUM
+            echo ${IROHA_VERSION:-$CIRCLE_BUILD_NUM} > version.txt
+            cd $IROHA_HOME
+      - run:
+          name: copy dockerfile to build dir
+          command: cp $IROHA_HOME/docker/release_dockerfile $IROHA_HOME/build/release_dockerfile.tmp
+      - store_artifacts:
+          path: build/iroha.deb
+          destination: iroha-linux.deb
+      - store_artifacts:
+          path: build/iroha.tar.gz
+          destination: iroha-linux.tar.gz
+      - store_artifacts:
+          path: build/version.txt
+          destination: version.txt
+      - persist_to_workspace:
+          root: build
+          paths:
+            - iroha.deb
+            - release_dockerfile.tmp
+
+
+
+  dockerize:
+    <<: *defaults
+    docker:
+      - image: hyperledger/iroha-docker-develop
+      - image: docker:17.05.0-ce-git
+    steps:
+      - checkout
```

Comment:
Checkout clones the project, so there is no need to make a copy of `release_dockerfile` to `build/release_dockerfile.tmp`.

Reply:
resolved